.github/workflows/integration_test.yml

```diff
@@ -108,6 +109,14 @@ jobs:
           IMAGE_TAG: "devel"
           CTR_CMD: docker
 
+      - name: Save artifacts
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: artifacts
+          path: ${{env.ARTIFACT_DIR}}
+          retention-days: 10
```

Comment:
We can decide whether we want to remove the retention for debug artifacts or extend the number of days.